### PR TITLE
Specs titles

### PR DESCRIPTION
**Pull request by Saimis777**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/1.7/back-office/shop-parameters/customer-settings/view-titles.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+👉 See the [specs of the add/edit social title]({{<ref "add-edit-social-titles.md">}} "Add/edit social titles") 
+
+**Delete.** The title can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Delete" button, the title is deleted, a successful message is displayed "Successful deletion."
+
+### Bulk actions 
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
```
Comment (fabienvallon) [inline content/1.7/back-office/shop-parameters/customer-settings/view-titles.md:65]: This checkbox behavoir looks weird to me

Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/view-titles.md:65]: @fabienvallon the checkbox for "Select all" stays selected as an image, but I believe that might be intentional to separate the images from selected and non selected, but work as intended.

Not sure though if that is the correct way? @MatShir

File: content/1.7/back-office/shop-parameters/customer-settings/add-edit-social-titles.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Has a tooltip: "Image width in pixels. Enter "0" to use the original size.
+
+5) Image height 
+
+Input field for the image height, by default has an input: "16". Only numbers are allowed in this section, when entering any other symbol, will show an error: "Width and height must be numeric values.".
+Has a tooltip: "Image height in pixels. Enter "0" to use the original size.
+
+### Call to action
+
+ - Cancel, when pressed redirects back to [specs of the view titles]({{<ref "view-titles.md">}} "view titles"), shows a message: "Successful update."
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/add-edit-social-titles.md]: @fabienvallon should this be raised as an issue or needs to be described?

File: content/1.7/back-office/shop-parameters/customer-settings/add-edit-social-titles.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+# Social titles
+
+In this section 5 fields are presented:
+
+1) Social title
+
+This is a mandatory input field, the field is multi language, can be translated to a selected language that is installed on the shop. Has a tooltip: "Invalid characters: 0-9!<>,;?=+()@#"{}_$%:". Trying to save with the field empty will show an error: "The field name is required at least in *default shop language*".
+Trying to save with invalid characters, will show an error (See issue [#28320](https://github.com/PrestaShop/PrestaShop/issues/28320)).
+The social title has a limit of 20 characters, if any more will be entered, when trying to save will show an error: "Your entry in field name exceeds max length 20 chars (incl. HTML tags).
```
[the remark this answers is not in the document]
Comment (Saimis777) [reply content/1.7/back-office/shop-parameters/customer-settings/add-edit-social-titles.md:14]: @fabienvallon could you clarify this part? Doesn't matter if the shop is multilanguage or only 1 is installed, the error states the same thing, only the selected language changes if different ones are used.

**Pull request merged**